doc/api/assert.md

```diff
@@ -161,6 +162,11 @@ assert.deepEqual(date, fakeDate);
 assert.deepStrictEqual(date, fakeDate);
 // AssertionError: 2017-03-11T14:25:31.849Z deepStrictEqual Date {}
 // Different type tags
+
+assert.deepStrictEqual({}, new Number(1));
+// Fails as the wrapped number is unwrapped and compared as well.
```

Comment:
Micro-nit: `Fails as the wrapped number is...` -> `Fails because the wrapped number is...`

Reply:
Thanks for the suggestion!